SS_write_report.tpl

```diff
@@ -1560,31 +1557,21 @@ FUNCTION void write_bigoutput()
       }
     }
   }
-  // REPORT_KEYWORD 17 SPR_SERIES
-  //  Fleet Fleet_Name Area Yr Era Seas Subseas Month Time
+  // REPORT_KEYWORD 17 SPR_SERIES  (equilibrium_SPR_and_YPR_calculations_for_each_year)
+  // 1=totbio, 2=smrybio, 3=smrynum, 4=enc_catch, 5=dead_catch, 6=ret_catch, 7=spbio, 8=recruit,
+  // 9=equ_totbio, 10=equ_smrybio, 11=equ_SSB_virgin, 12=equ_S1, 13=Gentime, 14=YPR, 15=meanage_spawners, 16=meanage_smrynums, 17=meanage_catch
+
   if (pick_report_use(17) == "Y")
   {
     SS2out << endl
-           << pick_report_name(17);
-    SS2out << "  uses_R0= " << Recr_virgin << endl
-           << "#NOTE: YPR_unit_is_Dead_Biomass" << endl;
-    SS2out << "Depletion_basis: " << depletion_basis << " # " << depletion_basis_label << endl;
-    SS2out << "F_report_basis: " << F_reporting << " # " << F_report_label << endl;
-    SS2out << "SPR_report_basis: " << SPR_reporting << " # " << SPR_report_label << endl;
-    // note  GENTIME is mean age of spawners weighted by fec(a)
-    SS2out << "Yr Era Bio_all Bio_Smry SSBzero SSBfished SSBfished/R SPR SPR_report YPR GenTime Deplete F_report"
-           << " Actual: Bio_all Bio_Smry Num_Smry MnAge_Smry Enc_Catch Dead_Catch Retain_Catch MnAge_Catch SSB Recruits Tot_Exploit"
-           << " More_F(by_Morph): ";
-    for (g = 1; g <= gmorph; g++)
-    {
-      SS2out << " aveF_" << g;
-    }
-    for (g = 1; g <= gmorph; g++)
-    {
-      SS2out << " maxF_" << g;
-    }
-    SS2out << " Enc_Catch_B Dead_Catch_B Retain_Catch_B  Enc_Catch_N Dead_Catch_N Retain_Catch_N sum_Apical_F F=Z-M  M";
-    SS2out << endl;
+           << pick_report_name(17) << endl;
+    SS2out << "#_NOTE:_reports_per_recruit_quantities_using_current_year_biology;_using_same_equil_calc_routine_used_for_reference_points" << endl;
+    SS2out << "#_NOTE:_current_year_biology_is_current_at-age_biology_from_time_series;_not_recalc_biology_from_current_growth_parameters" << endl;
+    SS2out << "#_NOTE:_uses_R0= " << Recr_virgin << endl;
+    SS2out << "#_NOTE:_Y/R_unit_is_Dead_Biomass" << endl;
+    SS2out << "#_NOTE:_gentime_is_mean_age_of_female_spawners_weighted_by_reproductive_value-at-age_(fec(g))" << endl;
+
+    SS2out << "Yr Era Bio_all_eq Bio_Smry_eq SSB_unfished_eq SSBfished_eq SSBfished/R SPR  Y/R GenTime" << endl;
```

Comment:
Is there a reason to change YPR to Y/R? It seems like YPR is used most other places.

Reply:
True, but I was striving for consistency with SSB/R.  Looked at other instances and agrtee with you, so will change back.